var/spack/repos/builtin/packages/libint/package.py

```diff
@@ -25,25 +25,27 @@
 from spack import *
 
 
-class Libint(Package):
+class Libint(AutotoolsPackage):
     """Libint is a high-performance library for computing
-    Gaussian integrals in quantum mechanics."""
+    Gaussian integrals in quantum mechanics.
+    """
 
     homepage = "https://github.com/evaleev/libint"
-    url      = "https://github.com/evaleev/libint/archive/v2.1.0.tar.gz"
+    url = "https://github.com/evaleev/libint/archive/v2.1.0.tar.gz"
 
+    version('2.2.0', 'da37dab862fb0b97a7ed7d007695ef47')
     version('2.1.0', 'd0dcb985fe32ddebc78fe571ce37e2d6')
     version('1.1.6', '990f67b55f49ecc18f32c58da9240684')
     version('1.1.5', '379b7d0718ff398715d6898807adf628')
 
     # Build dependencies
     depends_on('autoconf@2.52:', type='build')
-    depends_on('automake',       type='build')
-    depends_on('libtool',        type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
```

Comment:
I think @alalazo and I are the biggest perpetrators of the "edit wars". @alalazo uses `autopep8` to format everything consistently. This makes it easy to reformat a big package and ensure that it complies with `flake8`. I like to line things up because I think it makes it easier for humans to read. Neither of us are really "right". Enforcing `flake8` has made the "edit wars" much more benign than they used to be. If we could make flake8 even stricter it would prevent this, but I'm not going to vote for that because then I wouldn't be able to line things up anymore 😝 

Basically, I don't mind reviewing simple changes like this in a PR as long as others don't mind if I line things up once in a while.